egs2/TEMPLATE/speechlm1/speechlm.sh

```diff
@@ -822,7 +825,7 @@ if ! "${skip_eval}"; then
                 --nj ${nj} \
                 --inference_nj ${inference_nj} \
                 --gpu_inference ${gpu_inference} \
-                --nbest ${nbest}
+                --nbest ${nbest} ${scoring_args}
```

Comment:
@jctian98 I intentionally add this one to incorporate scoring related arguments for different tasks.